### PR TITLE
Datasets - Added Chicago Taxi Trips dataset

### PR DESCRIPTION
**Pull request by Ark-kun**

### CODE REVIEW

**Comment by numerology** [comment]
Do you mind elaborating on the reason why we need this?

**Comment by Ark-kun** [comment]
This component allows querying the full official and up to date dataset straight from the official governement source. There are some copies laying around, but they are often outdated. With this, you get the up to date information.

**Comment by numerology** [comment]
> This component allows querying the full official and up to date dataset straight from the official governement source. There are some copies laying around, but they are often outdated. With this, you get the up to date information.

Thanks! This sounds good to me. Then why is this in a hidden namespace?

File: components/_datasets/Chicago Taxi/components.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Chicago Taxi Trips dataset
```
Comment (numerology) [inline components/_datasets/Chicago Taxi/components.yaml:1]: I think it worth a .md doc to briefly introduce how to use this. Especially how to map each parameter to corresponding API query syntax. WDYT?

**Comment by Ark-kun** [comment]
> Thanks! This sounds good to me. Then why is this in a hidden namespace?

Oh. It's an interesting perspective. I did not consider it to be hidden namespace, more like an important group. Some other groups I envisioned are `_visualizers` and `_converters`. I just thought that the "datasets" group should be higher[-grouping-level] than, say, "arena" or "nuclio".

I can rename the directory to just be "datasets".

File: components/_datasets/Chicago Taxi/components.yaml (hunk 1 above)
Comment (Ark-kun) [reply components/_datasets/Chicago Taxi/components.yaml:1]: I'll do this.
I'll also create a sample that uses this component.

Comment (Ark-kun) [reply components/_datasets/Chicago Taxi/components.yaml]: For now I've added extended information about the component and query syntax to the description of the component.

**Comment by Ark-kun** [comment]
> I can rename the directory to just be "datasets".

Done.

**Pull request merged**